src/app/(proper_react)/(redesign)/(public)/LandingViewRedesign/components/CtaInputBanner.tsx

```diff
@@ -28,7 +28,8 @@ export const CtaInputBanner = (props: LandingPageProps) => {
             )}
           </p>
         </div>
-        {props.eligibleForPremium && props.scanLimitReached ? (
+        {props.enabledFeatureFlags.includes("DisableOneRepScans") ||
```

Comment:
Same as before re: logic outside US.

Edit: seeing more below, so maybe just do a full pass through the PR for it :)